README.md

```diff
@@ -730,6 +744,11 @@ An order for placing the rule in `pg_hba.conf`. Defaults to `150`.
 ####`target`
 This provides the target for the rule, and is generally an internal only property. Use with caution.
 
+####`postgresql_version`
+Defaults to the version set in `postgresql::server`.  Use this if you want to manage `pg_hba.conf` without managing the entire postgreSQL instance.
+
+####`manage_pg_hba_conf`
```

Comment:
I understand that you want to elide the inclusion, but I don't understand the usecase of "I have not included `postgresql::server` but I want to set `manage_pg_hba_conf` to `false` to cause a compile time error". So what seems to remain is avoiding referencing a undefined variable? Using `defined()` or `pick()` to do so would reduce the required API surface.